docs/static/logging.asciidoc

```diff
@@ -48,6 +48,77 @@ The logger is usually identified by a Java class name, such as
 path as in `org.logstash.dissect`.  For Ruby classes, like `LogStash::Outputs::Elasticsearch`,
 the logger name is obtained by lowercasing the full class name and replacing double colons with a single dot.
 
+NOTE: Consider using the default log4j configuration that is shipped with {ls}, as it is configured to work well for most deployments.  
+The next section describes how the rolling strategy works in case you need to make adjustments.
+
+[[rollover]]
+===== Rollover settings
+
+The `log4j2.properties` file has three appenders for writing to log files: 
+one for plain text, one with json format, and one to split log lines on per pipeline basis when you set the `pipeline.separate_logs` value.
+
+These appenders define: 
+
+* **triggering policies** that determine _if_ a rollover should be performed, and 
+* **rollover strategy**  to defines _how_ the rollover should be done.
+
+By default, two triggering policies are defined--time and size.
+
+* The **time** policy creates one file per day.
+* The **size** policy forces the creation of a new file after the file size surpasses 100 MB.
+
+The default strategy also performs file rollovers based on a maximum number files.
+When the limit of 30 files has been reached, the first (oldest) file is removed to create space for the new file.
+Subsequent files are renumbered accordingly. 
+
+Each file has a date, and files older than 7 days (default) are removed during rollover.
```

Comment:
@andsel If I'm understanding we're saying that we're showing users how to override default settings, but in the example, we're showing default settings.  Is that correct? 

Can we offer any guidelines on appropriate settings other than recommending that people don't change them?

Reply:
The intention is to describe what the default settings are and what they accomplish. The second example demonstrates how to switch the strategy to be size-based (the overall file sizes) instead of time-based (the default configuration that roll over after 7 days).